docs/backend-contracts.md

```diff
@@ -0,0 +1,19 @@
+# Table-centric documentation
```

Comment:
suggestion (non-blocking): This page will be visible to someone browsing the docs. So, perhaps adding an initial, maybe almost a placeholder, section that indicates:

* who this documentation page is aimed at
* what the page describes
* that this documentation page will be frequently changed

Something like (rough pass, wording could be improved; I'm not sure if we're capitalising "Backend Contracts" everywhere or not):

```markdown
# Introduction to Backend Contracts

This documentation on OpenSAFELY's Backend Contracts is a reference for data providers who wish to integrate their backend systems into OpenSAFELY.

Backend Contracts are an OpenSAFELY specification. By satisfying some or all of these Contracts, backend providers can certify their datasets for suitability with OpenSAFELY.

!!! warning

    This specification is currently under rapid development and is subject to frequent change.
```

Doesn't necessarily have to be in this PR though.

Reply:
Yep, fine!

Reply:
Oh, just to mention, because I wasn't sure and tested. It's possible to have the new `.md` file unlisted in the table of contents, but still be included in the build, if you know the URL. (That works even with `mkdocs build --strict`, which surprised me a bit.)

Maybe useful in future, if you want to have it previewed on the main docs site for people, but not yet "public".